packages/date-picker/src/DatePicker.tsx

```diff
@@ -19,38 +19,34 @@ export const DatePickerWrapper: React.FunctionComponent<DatePickerProps> = ({
   onDayChange,
   labelText,
   isDisabled = false,
-}) => (
-  <>
-    {labelText && <Label disabled={isDisabled} labelText={labelText} />}
-    <div
-      className={classnames(
-        styles.wrapper,
-        styles.withStartIconAdornment,
-        styles.withSearch,
-        {
-          [styles.withDisabled]: isDisabled,
+}) => {
+  const getNavbar = ({ ...navbarProps }: CalendarNavProps) => (
+    <CalendarNav {...navbarProps} />
+  )
+
+  return (
+    <>
+      <TextField
+        id="date-picker"
+        labelText={labelText}
+        icon={dateStart}
+        inputValue={
+          selectedDate ? selectedDate.toLocaleDateString() : undefined
         }
-      )}
-    >
-      <div className={styles.startIconAdornment}>
-        <Icon icon={dateStart} role="presentation" />
-      </div>
-      <DayPickerInput
-        value={selectedDate}
-        onDayChange={onDayChange}
-        dayPickerProps={{
-          selectedDays: selectedDate,
-          disabledDays: {
-            daysOfWeek: [0, 6],
-          },
-        }}
-        inputProps={{
-          disabled: isDisabled,
-          className: classnames(styles.input, {
-            [styles.disabled]: isDisabled,
-          }),
+        disabled={isDisabled}
+      />
+
+      <DayPicker
+        selectedDays={selectedDate}
+        // TODO create disabledDays prop, this is to display disabled styling
```

Comment:
Is this TODO referring to do away with the "magic numbers" and accept a descriptive string instead? (e.g. `mon/tue/wed/...`)?

Reply:
Correct! That's in the next card.